cross-dom-bridge/index.js

```diff
@@ -47,8 +47,8 @@ const setup = async() => {
       l1SignerOrProvider: l1Signer,
       l2SignerOrProvider: l2Signer
   })
-  l1ERC20 = new ethers.Contract(daiAddrs.l1Addr, erc20ABI, l1Signer)
-  l2ERC20 = new ethers.Contract(daiAddrs.l2Addr, erc20ABI, l2Signer)
+//  l1ERC20 = new ethers.Contract(daiAddrs.l1Addr, erc20ABI, l1Signer)
+//  l2ERC20 = new ethers.Contract(daiAddrs.l2Addr, erc20ABI, l2Signer)
```

Comment:
why is this commented out?

Reply:
Because I don't have an ERC-20 to bridge between Goerli and Optimism Goerli. I plan on rewriting std-bridge-custom-token and doing it with OUT, I just haven't yet.